public/static/locales/en/common.json

```diff
@@ -30,6 +33,7 @@
     "noOptions": "No results to display.",
     "openDrawer": "Open Drawer",
     "ok": "OK",
+    "polices": "Polices",
```

Comment:
😄 I didn't notice this when reviewing the screenshots, but this should be `Policies`

Reply:
`i` catching!